tsdb/series_partition.go

```diff
@@ -88,7 +88,7 @@ func (p *SeriesPartition) Open() error {
 		p.index = NewSeriesIndex(p.IndexPath())
 		if err := p.index.Open(); err != nil {
 			return err
-		} else if p.index.Recover(p.segments); err != nil {
+		} else if err := p.index.Recover(p.segments); err != nil {
```

Comment:
Here is nothing to do with the PR. As this is such a small typo, I added it incidentally.

Reply:
Thank you!